ShopAppTests/App/Account/Controllers/OrderDetailsViewControllerSpec.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  OrderDetailsViewControllerSpec.swift
+//  ShopAppTests
+//
+//  Created by Evgeniy Antonov on 2/21/18.
+//  Copyright © 2018 RubyGarage. All rights reserved.
+//
+
+import Nimble
+import Quick
+import RxSwift
+
+@testable import ShopApp
+
+class OrderDetailsViewControllerSpec: QuickSpec {
+    private let numberOfSections = 3
+    
+    override func spec() {
+        var viewController: OrderDetailsViewController!
+        var navigationController: NavigationController!
+        var tableProvider: OrderDetailsTableProvider!
+        var viewModel: OrderDetailsViewModelMock!
+        
+        beforeEach {
+            viewController = UIStoryboard(name: StoryboardNames.account, bundle: nil).instantiateViewController(withIdentifier: ControllerIdentifiers.orderDetails) as! OrderDetailsViewController
+            viewController.orderId = "order id"
+            let repository = OrderRepositoryMock()
+            let orderUseCaseMock = OrderUseCaseMock(repository: repository)
+            viewModel = OrderDetailsViewModelMock(orderUseCase: orderUseCaseMock)
+            viewController.viewModel = viewModel
+            tableProvider = OrderDetailsTableProvider()
+            viewController.tableProvider = tableProvider
+            navigationController = NavigationController(rootViewController: UIViewController())
+            navigationController.pushViewController(viewController, animated: false)
+            _ = viewController.view
+        }
+        
+        describe("when view loaded") {
+            it("should have view model with correct type") {
+                expect(viewController.viewModel).to(beAKindOf(OrderDetailsViewModel.self))
+            }
+            
+            it("should have correct title") {
+                expect(viewController.title) == "ControllerTitle.OrderDetails".localizable
+            }
+            
+            it("should have a back button") {
+                expect(viewController.navigationItem.leftBarButtonItem?.image) == #imageLiteral(resourceName: "arrow_left")
+            }
```

Comment:
Also need to check:
- `tableView.contentInset = TableView.defaultContentInsets`
- `viewModel.orderId = orderId`